docs/index.rst

```diff
@@ -12,6 +12,24 @@ The Roman Space Telescope Calibration Pipeline
    :align: center
    :alt: Nancy Roman Space Telescope
 
+Welcome to the documentation for the Roman calibration software,
+`romancal`. This package contains the Python software suite for the
+Roman Space Telescope (RST) calibration pipeline, which processes data
+from all Roman instruments by applying various corrections to produce
+science-ready, calibrated output products including fully calibrated
+individual exposures as well as high-level data products (mosaics,
+catalogs, etc.). The tools in this package allow users to run and
+configure the pipeline to custom process their Roman data.
+Additionally, the `romancal` package contains the interface to
```

Comment:
Again `romancal` is in back ticks, which attempt to create a link but the link is not provided.

Reply:
Fixed.